app/src/scenes/auth/signup.js

```diff
@@ -24,7 +24,8 @@ export default () => {
         initialValues={{ username: "", organisation: "", password: "" }}
         onSubmit={async (values, actions) => {
           try {
-            const { user, token } = await api.post(`/user/signup`, values);
+            const { user, token, code } = await api.post(`/user/signup`, values);
```

Comment:
what is code in this context? why is this a bug?

Reply:
I reviewed it.
it's not a bug.
code is having value as "PASSWORD_VALIDATION_FAILED" when user signup with alphabets only password.
however, I should have used "ok" to popup error in UI, when signup fails